scripts/combine/setupCombine.py

```diff
@@ -345,35 +348,78 @@ def assertSample(name, startsWith=["W", "Z"], excludeMatch=[]):
     if args.poiAsNoi:
         noi_args = dict(
             group=f"normXsec{label}",
-            scale=1 if args.priorNormXsec < 0 else args.priorNormXsec, # histogram represents an (args.priorNormXsec*100)% prior
-            mirror=True,
             passToFakes=passSystToFakes,
         )
         if args.theoryAgnostic:
-            nuisanceBaseName = f"norm{label}CHANNEL_"
+            # open file with theory bands
+            with h5py.File(f"{common.data_dir}/angularCoefficients/theoryband_variations.hdf5", "r") as ff:
+                scale_hists = narf.ioutils.pickle_load_h5py(ff["theorybands"])
             # First do in acceptance bins, then OOA later (for OOA we need to group bins into macro regions)
+            nuisanceBaseName = f"norm{label}_"
             cardTool.addSystematic("yieldsTheoryAgnostic",
+                                   rename=f"normWplus",
                                    **noi_args,
-                                   systAxes=poi_axes, 
+                                   mirror=False,
+                                   systAxes=poi_axes+["downUpVar"],
                                    processes=["signal_samples"],
-                                   baseName=nuisanceBaseName,
+                                   baseName=f"normWplus_",
                                    noConstraint=True if args.priorNormXsec < 0 else False,
+                                   scale=1,
+                                   formatWithValue=[None,None,"low",None],
                                    #customizeNuisanceAttributes={".*AngCoeff4" : {"scale" : 1, "shapeType": "shapeNoConstraint"}},
-                                   labelsByAxis=["PtVBin", "YVBin", "AngCoeff"],
+                                   labelsByAxis=["PtV", "YVBin", "Helicity","downUpVar"],
                                    systAxesFlow=[], # only bins in acceptance in this call
                                    skipEntries=[{"helicitySig" : [6,7,8]}], # removing last three indices out of 9 (0,1,...,7,8) corresponding to A5,6,7
                                    actionMap={
-                                       m.name: (lambda h: hh.addHists(h[{ax: hist.tag.Slicer()[::hist.sum] for ax in poi_axes}], h, scale2=args.scaleNormXsecHistYields))
-                                       for g in cardTool.procGroups["signal_samples"] for m in cardTool.datagroups.groups[g].members},
+                                    m.name: (lambda h, scale_hist=scale_hists[m.name]: hh.addHists(h[{ax: hist.tag.Slicer()[::hist.sum] for ax in poi_axes}], hh.multiplyHists(hh.addGenericAxis(h,common.down_up_axis), hh.rescaleBandVariation(scale_hist,args.theoryAgnosticBandSize),flow=False), scale2=args.scaleNormXsecHistYields)) if "plus" in m.name else (lambda h: h[{ax: hist.tag.Slicer()[::hist.sum] for ax in poi_axes}]) for g in cardTool.procGroups["signal_samples"] for m in cardTool.datagroups.groups[g].members},
```

Comment:
Do you really need to define the up/down variations here? Shouldn't it be equivalent to mirroring later ? Perhaps this is just how you define scale_hist from the input file.
Also, I might misunderstand what happens, but aren't args.scaleNormXsecHistYields and args. args.theoryAgnosticBandSize trying to achieve a similar thing in this new context? I don't think one can use these two options together (or at least one would lose track of what the final variation actually is, since you would be multiplying two things scaling one factor in the product, then you scale again the whole product).
So I suspect that at this point scaleNormXsecHistYields becomes useless (in which case it might be removed, or if they do different things and are not supposed to work together there should be a protection).

Reply:
Yes the bands are asymmetric and that's what we want, and this is fully supported by combinetf. I have removed scaleNormXsecHistYields from that line because as pointed above it doesn't make sense anymore, because we need additive scaling before multiplying.